src/main/java/com/example/teamboolean/apprentidash/ApprentiDashController.java

```diff
@@ -253,8 +262,6 @@ public String deleteDay(@PathVariable long dayId, Principal p){
 
```

Comment:
For the case where the day isn't found in the list, we probably want to pass a message to the error page so the user knows what happened.